packages/terra-status/src/Status.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
 import styles from './Status.module.scss';
+import VisuallyHiddenText from '../../terra-visually-hidden-text';
```

Comment:
We should be importing the module directly from `terra-visually-hidden-text`. This works fine in the terra-core repo because `terra-visually-hidden-text` is relative to `terra-status`, however, if `terra-status` was released & imported by a different project, this would be broken.

Reply:
You'll also need to add `terra-visually-hidden-text` to this component's package.json file as a depdency.